### ISSUE TITLE
Swal.IsLoadingAsync() is broken

### ISSUE THREAD

**Issue by ohenebaamissah** [Basaingeal/Razor.SweetAlert2#390]
Swal.IsLoading is not exposed as a method on the window object.

![Swal IsLoadingAsync()](https://user-images.githubusercontent.com/22178600/70310794-64502380-1808-11ea-8510-03b8977c9823.png)

Possibly due to a typo in SweetAlert.ts

```ts
razorSwal.HideLoading = (): void => {
  Swal.hideLoading();
};

razorSwal.HideLoading = (): boolean => {
  return Swal.isLoading();
};
```

**Comment by Basaingeal** [comment]
Yup. I think you're right. I'll try to have a fix out in the next few hours.

**Comment by Basaingeal** [comment]
This should be fixed in `v2.2.1` which will be available on Nuget shortly.
Thanks so much for reporting. Always glad to see people finding the library useful.
